storage/local/storage.go

```diff
@@ -372,6 +379,16 @@ func (s *MemorySeriesStorage) Start() (err error) {
 
 	log.Info("Loading series map and head chunks...")
 	s.fpToSeries, s.numChunksToPersist, err = p.loadSeriesMapAndHeads()
+	openHeadCount := 0
+	for fp := range s.fpToSeries.fpIter() {
+		if series, ok := s.fpToSeries.get(fp); ok {
+			if !series.headChunkClosed {
+				openHeadCount++
```

Comment:
You can `inc()` s.numHeadChunks directly here. It's very fast compared to the iteration above.

(That would be another concern: For best performance, it would be better to count the open head chunks during loading, but since this is a one-off at start time, it probably doesn't matter much.